scripts/compile_cost_assumptions.py

```diff
@@ -1675,6 +1902,9 @@ def add_mean_solar_rooftop(data):
         costs = adjust_for_inflation(costs, new_format_without_solar, 2020)
         # add desalination and clean water tank storage
         costs = add_desalinsation_data(costs)
+        # add energy storage database
+        if snakemake.config['parzen_energy_storage']:
+            costs = add_energy_storage_database(costs, year)
```

Comment:
your costs should also be adjusted to inflation, see function above

Reply:
added

Reply:
The config.yaml now has the option `approx_beyond_2030` with two options `"geometric_series"` (optimistic that technologies will improve) and `"same_as_2030"` (very pessimistic assuming costs are not decreasing after 2030). Below I am also just showing briefly that the options work fine.

![image](https://github.com/PyPSA/technology-data/assets/61968949/49396235-cc69-4796-8620-4ee77826eb47)

# Option "same_as_2030"
## 2020 results
![image](https://github.com/PyPSA/technology-data/assets/61968949/19b3d2b7-cac3-484b-9d50-5c17e87d698d)
## 2030 results
![image](https://github.com/PyPSA/technology-data/assets/61968949/10e86959-79b4-4aca-b8df-ff0220ece85b)
## 2050 results
![image](https://github.com/PyPSA/technology-data/assets/61968949/4c0959fd-21b3-4f9c-b85f-3e5985c0cb43)


# Option "geometric_series"
## 2050 results
![image](https://github.com/PyPSA/technology-data/assets/61968949/f396c967-7207-4aa0-826c-8711821c2e7b)

Reply:
![image](https://github.com/PyPSA/technology-data/assets/61968949/d6e1a210-4816-492e-8543-95d30a270fb0)